### PR TITLE
Update seednode/pricenode install scripts

### PR DESCRIPTION
**Pull request by devinbileck**
While deploying a new seednode instance on Ubuntu 20.04, I ran into a few issues with the install script and README. This PR addresses those issues.

### CODE REVIEW

**Review by cbeams — APPROVED** [review]
utACK. Thanks, Devin!

Before merging, I'd recommend at least one of the other @bisq-network/pricenode-operators try these changes out for themselves.

**Comment by ripcurlx** [comment]
@wiz Could you have a look at this changes before I merge this based on https://github.com/bisq-network/bisq/pull/4319#pullrequestreview-433045580

**Comment by wiz** [comment]
utACK
Devin, thanks for fixing this!

File: pricenode/install_pricenode_debian.sh [hunk 1]
```diff
@@ -34,6 +34,11 @@ echo "[*] Upgrading apt packages"
 sudo -H -i -u "${ROOT_USER}" DEBIAN_FRONTEND=noninteractive apt-get update -q
 sudo -H -i -u "${ROOT_USER}" DEBIAN_FRONTEND=noninteractive apt-get upgrade -qq -y
 
+echo "[*] Installing Git LFS"
+sudo -H -i -u "${ROOT_USER}" curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```
Comment (wiz) [inline pricenode/install_pricenode_debian.sh:38]: Why is this needed? Doesn't ubuntu have a git-lfs package by default?

Comment (Emzy) [reply pricenode/install_pricenode_debian.sh:38]: I also think ```apt install git-lfs``` should work just fine. No need for external packages.

Comment (devinbileck) [reply pricenode/install_pricenode_debian.sh:38]: This is used to ensure the latest git-lfs package is installed. I don't recall now, but perhaps the latest isn't necessary for our needs and it should be fine to remove this line.
https://github.com/git-lfs/git-lfs/wiki/Installation#debian-and-ubuntu
